demisto_sdk/commands/setup_env/setup_environment.py

```diff
@@ -491,23 +492,56 @@ def install_virtualenv(
     logger.info(f"Creating virtualenv for {integration_script.name}")
     shutil.rmtree(venv_path, ignore_errors=True)
     venv.create(venv_path, with_pip=True)
-    for req in requirements:
-        if not req:
-            continue
-        try:
-            subprocess.run(
-                [
-                    f"{venv_path / 'bin' / 'pip'}",
-                    "-q",
-                    "--disable-pip-version-check",
-                    "install",
-                    req,
-                ],
-                check=True,
-            )
-            logger.info(f"Installed {req}")
-        except subprocess.CalledProcessError:
-            logger.warning(f"Could not install {req}, skipping...")
+    env = os.environ.copy()
+    env.update({"VIRTUAL_ENV": str(venv_path)})
+    requirements = [req for req in requirements if req]
+    try:
+        subprocess.run(
```

Comment:
for readability, let's split to methods: `install_all_using_uv`, `install_single_using_uv`, `install_all_using_pip`